src/dune_rules/scope.ml

```diff
@@ -4,14 +4,14 @@ open Memo.O
 type t =
   { project : Dune_project.t
   ; db : Lib.DB.t
-  ; coq_db : Coq_lib.DB.t Memo.Lazy.t
+  ; coq_db : Coq_lib.DB.t Memo.t
```

Comment:
Since this is no longer lazy, wouldn't this mean that we construct the coq_db even if it is not being used?

Reply:
It's still lazy. This `Memo.t` is just forcing the cell constructed in `Coq_scope.t`

Reply:
The work that is no longer saved is the map lookup in the scope database. However such a map lookup has less overhead than memo's machinery so there's no issue.